scripts/rook.sh

```diff
@@ -83,6 +83,7 @@ kubectl_retry() {
 function deploy_rook() {
         kubectl_retry create -f "${ROOK_URL}/common.yaml"
         kubectl_retry create -f "${ROOK_URL}/operator.yaml"
+        kubectl_retry create -f "${ROOK_URL}/crds.yaml"
```

Comment:
afaik, crds.yaml runs before operator.yaml, or else it fails with CRD not found

Reply:
apply CRD only if rook version >= 1.5